src/utils/margins/margins.ts

```diff
@@ -1,12 +1,12 @@
-const integerVals = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10
-const autoVals = 'auto'
+import type { integers } from '../lib/integers'
 
```

Comment:
this type was really ending up as 'number' or 'string', now only 1-10 or auto will be allowed